README.md

```diff
@@ -1,7 +1,7 @@
-Truesight pulse Process Plugin
----------------------------------
+Meter Process Plugin
```

Comment:
We should have it as "TrueSight Pulse Process Plugin" here and not "Meter Process Plugin"